docs/manuals/core/scheduling/index.md

```diff
@@ -0,0 +1,278 @@
+# Taipy Scheduler
+
+!!! warning "Available in Taipy Enterprise edition"
+
+    This section is relevant only to the Enterprise edition of Taipy.
+
+This documentation focuses on providing necessary information to use the Taipy `Scheduler^`, and in particular
+the capabilities related to scenario management and job execution. It is assumed that the reader already knows the
```

Comment:
```suggestion
This documentation is primarily aimed at giving you the essential information needed to utilize the Taipy `Scheduler^`, with a specific emphasis on its capabilities regarding scenario management and job execution. It is assumed that the reader already knows the
```

Reply:
PLease make the lines less than 100 char.

Reply:
I feel like the sentence is using too many advance words.

Tbh, I don't know what "with a specific emphasis on its capabilities regarding scenario management and job execution" means without re-reading it 2-3 times.

Reply:
OK maybe let's try this less advanced wording:
This documentation is centered around providing essential information for using the Taipy `Scheduler^`, 
particularly focusing on the capabilities related to scenario management and job execution. 
It is assumed that the reader already knows the

Reply:
this is reasonable IMO. What do you think @jrobinAV?